core/src/main/java/com/fightpandemics/core/data/model/userlocationdetails/Location.kt

```diff
@@ -3,10 +3,10 @@ package com.fightpandemics.core.data.model.userlocationdetails
 import com.squareup.moshi.Json
 
 data class Location(
-    @field:Json(name = "address") val address: String,
-    @field:Json(name = "city") val city: String,
-    @field:Json(name = "coordinates") val coordinates: List<Double>,
-    @field:Json(name = "country") val country: String,
-    @field:Json(name = "state") val state: String,
-    @field:Json(name = "zip") val zip: String
+    @field:Json(name = "address") val address: String?,
```

Comment:
Why do these need to be nullable? Wouldn't it be better if they are empty strings if non-existent?

Reply:
If they are not nullable, that causes Moshi to crash the app since the location JSON may or may not have these specific fields. Also since they are nullable, later in the code with the '?:' operator they are being set to empty strings when the response might be null. For example, here:
![image](https://user-images.githubusercontent.com/36528546/109408066-e6ab9280-793a-11eb-9a14-e2b9c3f9f642.png)